lib/cassandra_migrations/cql-rb-wrapper.rb

```diff
@@ -34,13 +34,12 @@ def add(*args)
 
 class Client
   def self.connect(options)
-    if defined?(Spring)
-      Spring.after_fork do
-        @cluster = Cassandra.cluster(options)
-      end
-    else
-      @cluster = Cassandra.cluster(options)
+    @cluster = Cassandra.cluster(options)
```

Comment:
There's more to look at with this piece. I'm seeing some issues. Will look into it more tomorrow.